demos/embeddings/README.md

```diff
@@ -155,7 +155,39 @@ It will report results like `Similarity score as cos_sim 0.97654650115054`.
 
 ## Benchmarking feature extraction
 
-TBD
+An asynchronous benchmarking client can be use to access the model server performance with various load conditions:
+```bash
```

Comment:
From automatic tests point of view at this moment current directory is model_server.
I suggest to add **cd demos/embeddings** command here